### DISCUSSION TITLE
More colorful text for Medical Status HUD

### DISCUSSION THREAD

**Discussion by DrBruhman** [FakeFishGames/Barotrauma#15445] [Ideas and suggestions]
vanilla:
![398911583-0ad4a8e0-4511-49b0-a6ca-2657fe75306c](https://github.com/user-attachments/assets/a7f1759b-6f6e-4ce3-8af5-6cddc4067213)
modified:
![изображение](https://github.com/user-attachments/assets/bb079aa1-0638-4e67-8841-7d1046294c90)
https://github.com/FakeFishGames/Barotrauma/blob/38ca721cb0fd94aadbb9c1886ba2338bbd5f1180/Barotrauma/BarotraumaClient/ClientSource/Items/Components/StatusHUD.cs#L357
with use of this method instead of hardcoded color lerp
https://github.com/FakeFishGames/Barotrauma/blob/38ca721cb0fd94aadbb9c1886ba2338bbd5f1180/Barotrauma/BarotraumaClient/ClientSource/Characters/Health/CharacterHealth.cs#L1254

**Comment by girlyguppy** [comment]
this is peak
